### ISSUE TITLE
Insight on MAE pretraining for the entire model (encoder + decoder)

### ISSUE THREAD

**Issue by VGrondin** [facebookresearch/Mask2Former#144]
Hello, I find this work very interesting. I was wondering if you thought it possible to pretrain the entire Mask2Former model using a reconstruction loss of MAE, then finetuning it on an object detection task. This would have the advantage of not having to discard the decoder like in MAE. 

Thanks!

**Comment by bowenc0221** [comment]
Yes, I think it is worth a try!
